dcos/dcos-ui
# [spike] DCOS-53734: Update message catalogs

Update message catalogs with the latest changes.

Closes https://jira.mesosphere.com/browse/DCOS-53734

## Testing
1. Use the latest master from the other repository.
2. Run `npm run util:lingui:extract-with-plugins`.
3. Verify that the message catalogs didn't change.

## Trade-offs
None.

## Dependencies
None.

## Screenshots
None.
